test/team_statistics_test.rb

```diff
@@ -0,0 +1,95 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/stat_tracker"
+require "./lib/game_statistics"
+require "./lib/team_statistics"
+require "pry";
+
+class TeamStatisticsTest < Minitest::Test
+  def setup
+    game_path = './data/dummy_game.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './data/dummy_game_teams.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    @stat_tracker = StatTracker.from_csv(locations)
+    @game_statistics = GameStatistics.new(@stat_tracker)
+    @team_statistics = TeamStatistics.new(@stat_tracker)
+  end
+
+  # Each of the methods below take a team id as an argument. Using that team id,
```

Comment:
At the end we'll need to remove the comments and spaces if there are extra. No worries right now though!